src/layouts/ReviewRequest/components/RequestOverview/RequestOverview.tsx

```diff
@@ -337,26 +336,49 @@ export const RequestOverview = ({
       ),
     }),
     columnHelper.display({
-      id: "actions",
+      id: "action",
       header: () => (
         <Th borderBottom="1px solid" borderColor="gray.100" w="10rem" />
       ),
       cell: ({ row }) => (
         <HStack spacing="0.25rem">
           {allowEditing && (
-            <Link href="www.google.com">
+            <Link
```

Comment:
@seaerchin lgtm, could you add a screenshot of how this looks like on the UI or add steps to test this locally?

Reply:
there's a story written for this so it should show inside the storybook!